packages/yourloops/lib/team/team.hook.tsx

```diff
@@ -92,15 +94,19 @@ function TeamContextImpl(): TeamContext {
   }
 
   const getMedicalTeams = (): Team[] => {
-    return getTeamsByType([TeamType.medical])
+    return getTeamsByType(TeamType.medical)
+  }
+
+  const isPrivate = (team: Team): boolean => {
```

Comment:
**🎯 suggestion (blocking):** ‏This method has no place in the team hook as it is not dependant on any state. This should be in team util.